pkgs/by-name/pl/plymouth-matrix-theme/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  lib,
+  unstableGitUpdater,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "plymouth-matrix-theme";
+  version = "0.1.0-unstable-19-02-2017";
```

Comment:
Should be `YYYY-MM-DD`, not `DD-MM-YYYY`. Running the updateScript will fail because it thinks the version is different.

```
+ update-source-version plymouth-matrix-theme 0.1.0-unstable-2017-02-19 --rev=b2268f25dea7537ed5709b00d5a83b3600265c54
update-source-version: error: Failed to replace source revision 'b2268f25dea7537ed5709b00d5a83b3600265c54' to 'b2268f25dea7537ed5709b00d5a83b3600265c54' in 'plymouth-matrix-theme'!
```